test/utils.js

```diff
@@ -17,6 +17,9 @@ const parseSpec = (spec) => {
 };
 
 const findSpecLineIndex = (line, fileLines) => {
+  const mayBeFileError = line === 1;
```

Comment:
maye check fileLines count?

Reply:
The file can have multiple lines but the error still gets reported for the first one